acceptance/analyzer_test.go

```diff
@@ -54,6 +54,10 @@ func TestAnalyzer(t *testing.T) {
 	defer registry.Stop(t)
 
 	os.Setenv("DOCKER_CONFIG", registry.DockerDirectory)
+	// Copy docker config directory to analyze-image container
+	targetDockerConfig := filepath.Join("testdata", "analyzer", "analyze-image", "container", "docker-config")
+	h.AssertNil(t, os.RemoveAll(filepath.Join(targetDockerConfig, "config.json")))
+	h.RecursiveCopy(t, registry.DockerDirectory, targetDockerConfig)
```

Comment:
Now we can delete all occurrences of `"--env", "CNB_REGISTRY_AUTH={}",` in this file : )

Reply:
I thought that we already did it.
I'll delete it!